### ISSUE TITLE
Is the cheat engine source a private commit?

### ISSUE THREAD

**Issue by qwer0123456** [cheat-engine/cheat-engine#3049]
I checked the Patreon page and 7.6 beta is the latest
but the source has not been updated for N years from 7.5 stop
I have DBVM freezing and want to do small dbvm fix custom build

**Comment by cheat-engine** [comment]
yeah, certain groups have used ce's sourcecode without a license so i didn't feel like committing to the public git.

Maybe in the future again

**Comment by Symbai** [comment]
Well that makes sense and is fair enough but it doesn't explain why the latest compiled release is from Feb 2023. Almost 2 years ago. Cant you just at least update the compiled releases from time to time?

**Comment by cheat-engine** [comment]
hopefully there will be one soon

**Comment by BinToss** [comment]
@cheat-engine You may want to pin an issue for CE 7.6's release issues so you don't need to repeat yourself.

e.g. 
---

#### Why is the download on the website the wrong size/filename?

> The download provider sometimes has issues publishing new version and/or takes time to update the file. The filename may be outdated while the file's data is up-to-date.

#### Where is the source code for CE 7.6? Why is it private?

> The source code is currently private because the public source code had been copied without a license. (Speculation: malicious forks? CLA violations?). It might be published at a later date.

Related Issues:
- #3049
- #3109
- #3087

**Comment by Parsa307** [comment]
> yeah, certain groups have used ce's sourcecode without a license so i didn't feel like committing to the public git.
> 
> Maybe in the future again

Literally that's an excuse, who in the hell cares about seeing a skid kiddie removing a license or changing it? They never trend.

Publish the SRC.

We really don't want to deal with your Adware-Offerings.

Or at-least make it clear, your making the program Closed-Source.

We compile your program, for that reason.

We never rip you off. Is just skid kiddies ripping people.

**Comment by Twig6943** [comment]
@cheat-engine 

Was making the later versions source available (not open source) with a license [like this](https://github.com/stenzek/duckstation/blob/master/LICENSE) ever considered? This would be better than it being entirely closed source

(Don't want to spam/repeat whats been said but I don't think anybodys ever proposed this)

**Comment by Syzzle07** [comment]
Imagine if every single open source software would go closed source (and pretend not to be) because a minority (assuming this is even real) "removed muh license".
